packages/cli/src/commands/doctor/healthchecks/androidHomeEnvVariable.ts

```diff
@@ -18,15 +18,15 @@ const platform = process.platform as 'darwin' | 'win32' | 'linux';
 
 const message = `Read more about how to set the ${label} at ${chalk.dim(
   URLS[platform],
-)}.`;
+)}`;
 
 export default {
   label,
   getDiagnostics: async () => ({
     needsToBeFixed: !process.env.ANDROID_HOME,
   }),
   runAutomaticFix: async ({loader}: {loader: Ora}) => {
-    loader.info();
+    loader.fail();
```

Comment:
why logger.fail?

Reply:
Consistency, we can't fix that automatically.

The other health checks do a `fail` when logging how to manually fix it when they try to fix something automatically and it doesn't succeed, UI looks really strange with two messages that have the same content but with different loaders.

Reply:
Can't we move this call up? `runAutomaticFix` is a generic interface which we run in a single place, I think it just makes more sense to call `loader.fail()` once there, instead of thinking about it in every fix fn, or I don't see something?

Reply:
I think it was like that originally, I have changed it over time because the health check has to have full control of what is displayed on the screen once it's started IMO.

We could throw an error here and catch in on `doctor.ts` but I am not sure if that would be an improvement.